src/wallet/db.h

```diff
@@ -27,7 +27,7 @@ class CDBEnv
 private:
     bool fDbEnvInit;
     bool fMockDb;
-    boost::filesystem::path path;
+    std::string strPath;
```

Comment:
Please add a comment here why this field is a string instead of a `boost::filesystem::path`, so that a future smart-ass doesn't think "hey let's use that type directly".